webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -4722,7 +4722,14 @@ var userOverride = {
                                 _this.phoneNumberValidation
                             );
                     }
+                    function ValidateEmail(email) {
+                        var regexp = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```

Comment:
this same thing is not present in existing code base?

Reply:
Yes the same function was present, I have updated it.